app/plans/get.controller.js

```diff
@@ -0,0 +1,31 @@
+const displayText = require('./displayText')
+const { getSentencePlanSummary } = require('../../common/data/offenderSentencePlanSummary')
+
+const hasActivePlan = plans => {
+  if (isEmptyObject(plans)) return false
+  let activePlanPresent = false
+  plans.forEach(plan => {
+    if (!plan.completedDate) {
+      activePlanPresent = true
+    }
+  })
+  return activePlanPresent
+}
+
+function isEmptyObject(obj) {
```

Comment:
Feels like a generic toolkit utility function. Should we extract to a utility file maybe?

Reply:
Agreed